sdp/processors/datasets/mls/create_initial_manifest.py

```diff
@@ -39,12 +42,20 @@ class CreateInitialManifestMLS(BaseParallelProcessor):
             This will be used to format the URL from which we attempt to download the data.
             E.g., "english", "italian", "spanish", etc.
         data_split (str): "train", "dev" or "test".
-        resampled_audio_dir (str): the directory where the resampled
-            wav files will be stored.
-        target_samplerate (int): sample rate (Hz) to use for resampling.
+        resampled_audio_dir (str or None): if specified, the directory where the resampled
+            wav files will be stored. If not specified, the audio will not be resampled and
+            the parameters `target_samplerate` and `target_nchannels` will be ignored.
```

Comment:
Just a note that \`something\` results in italics in .rst files (which is what we generate for the API docs). If instead you intended this to be a "code block" then it should be \`\`something\`\`.